web/scripts/defaultGraph.js

```diff
@@ -1,119 +1,249 @@
 export const defaultGraph = {
-	last_node_id: 9,
-	last_link_id: 9,
+	last_node_id: 15,
+	last_link_id: 30,
```

Comment:
Why are we hard coding these values? @suchana-aiplanet

Reply:
This is just the default graph that loads on screen when user clicks on load default , gives the premade workflow to test the execution.